src/actions/index.js

```diff
@@ -0,0 +1,18 @@
+export const CREATE_BOOK = 'CREATE_BOOK';
+export const REMOVE_BOOK = 'REMOVE_BOOK';
+
+export const createBook = (
+  book = {
+    id: Math.floor(Math.random() * 1000),
+    title: 'New Book',
+    category: null,
+  },
```

Comment:
- [ ] Was this default value for the book object for testing purposes? We can remove that by now since your application should not trigger `createBook` action without providing book object